content/strategy-goals/strategy/code-insights/index.md

```diff
@@ -40,113 +41,109 @@ To deliver on our mission, we will:
 
 #### 1 Year vision
 
-> Any Sourcegraph user can create code insights for their code to answer any question that can be known with a Sourcegraph search or API query, and export/filter/find/receive this information wherever they need.
+> Any Sourcegraph user uses Code Insights-powered features to get a uniquely valuable view of code that displays actionable information about historical usage, code structure, or other high-level patterns that make their daily work more effective.
```

Comment:
```suggestion
> Any Sourcegraph user uses Code Insights and Code Insights-powered features to get a uniquely valuable view of code that displays actionable information about historical usage, code structure, or other high-level patterns that make their daily work more effective.
```

Reply:
@Joelkw I wouldn't so drastically move us away from the core insights.

Reply:
got it, will tweak